src/installation/guides/chroot.md

```diff
@@ -0,0 +1,302 @@
+This may be an existing installation of Void, an official [live
+image](../live-images/prep.md) or any Linux installation running a [statically
+linked XBPS](../../xbps/troubleshooting/static.md).
+
+The **ROOTFS method** requires only a host operating system that can enter a
+Linux chroot and has [tar(1)](https://man.voidlinux.org/tar.1) and
+[xz(1)](https://man.voidlinux.org/xz.1) installed. This method may be preferable
+if you wish to install Void using a different Linux distribution.
+
+## Prepare Filesystems
+
+[Partition your disks](../live-images/partitions.md) and format them using
+[mke2fs(8)](https://man.voidlinux.org/mke2fs.8),
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8),
+[mkfs.btrfs(8)](https://man.voidlinux.org/mkfs.btrfs.8) or whatever tools are
+necessary for your filesystem(s) of choice.
+
+[mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8) is also available to
+create FAT32 partitions. However, due to restrictions associated with FAT
+filesystems, it should only be used when no other filesystem is suitable (such
+as the EFI System Partition).
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) and
+[fdisk(8)](https://man.voidlinux.org/fdisk.8) are available on the live images
+for partitioning, but you may wish to use
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+For a UEFI booting system, make sure to create an EFI System Partition (ESP).
+The ESP should have the partition type "EFI System" (code `EF00`) and be
+formatted as fat32 using [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+If you're unsure what partitions to create, create a 1GB partition of type `EFI
+System` (code `EF00`) then create a second partition using the remainder of the
+drive with type `Linux Filesystem` (code `8300`).
+
+Format these partitions with fat32 and ext4, respectively.
+
+```
+# mkfs.vfat /dev/sda1
+# mkfs.ext4 /dev/sda2
+```
+
+### Create a New Root and Mount Filesystems
+
+This guide will assume the new root filesystem is mounted on `/mnt`. You may
+wish to mount it elsewhere.
+
+If using UEFI, mount the EFI System Partition as `/mnt/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `dev/sda1` is the EFI
+System Partition:
+
+```
+# mount /dev/sda2 /mnt/
+# mkdir -p /mnt/boot/efi/
+# mount /dev/sda1 /mnt/boot/efi/
+```
+
+Initialize swap space, if desired, using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+Follow only one of the two following subsections.
+
+### The XBPS Method
+
+[Select an XBPS mirror](../../xbps/repositories/mirrors/index.md). For
+simplicity, save this URL to a shell variable for later use. Append `/current`
+to this repository URL to install a glibc distribution, or `/current/musl` for a
+musl distribution.
+
+Note: musl binaries are not available for i686.
+
+For example, this is the URL for glibc from the Tier 1 server in Finland:
```

Comment:
"from a Tier 1 server:" - if the location changes, we don't want to have to go through the docs as a whole.

Reply:
you're okay keeping the url here though?

Reply:
Sure!